services/offline/src/setupRTL.js

```diff
@@ -1,5 +1 @@
 import '@testing-library/jest-dom/extend-expect'
-
-process.on('unhandledRejection', err => {
-    throw err
-})
```

Comment:
Removed this because it made it quite hard to debug tests for this largely asynchronous code